test/Microsoft.AspNetCore.Mvc.FunctionalTests/HtmlGenerationWithCultureTest.cs

```diff
@@ -149,6 +152,14 @@ public async Task CacheTagHelper_VaryByCultureComposesWithOtherVaryByOptions()
             // Assert - 3
             Assert.Equal("fr-FR", culture);
             Assert.Equal("14", correlationId);
+
+            if (correlationId != "10")
+            {
+                // This is logging to investigate potential flakyness in this test tracked by https://github.com/aspnet/Mvc/issues/8281
```

Comment:
Your spelling is flaky: https://www.thefreedictionary.com/flakiness

Reply:
😆

Reply:
Think you need this to be `cachedCorrelationId`?